dijet/config/cutflow_variables.py

```diff
@@ -66,11 +64,11 @@ def add_cutflow_variables(config: od.Config) -> None:
     #         binning=(11, -0.5, 10.5),
     #         x_title=f"Number of {obj}s",
     #     )
-    
+
     for prop in ("asymmetry", "alpha", "pt_avg"):
         config.add_variable(
             name=f"cf_dijets_{prop}",
             expression=f"cutflow.dijets_{prop}",
             binning=var_binning[prop],
-            x_title=var_title_format[prop]
-        )
\ No newline at end of file
+            x_title=var_title_format[prop],
```

Comment:
You could add the unit here, if one is defined in `var_unit` above:
```suggestion
            x_title=var_title_format[prop],
            unit=var_unit.get(prop),
```